stdlib/public/core/SmallString.swift

```diff
@@ -105,13 +105,9 @@ extension _SmallString {
 
   // Give raw, nul-terminated code units. This is only for limited internal
   // usage: it always clears the discriminator and count (in case it's full)
-  @inlinable
+  @inlinable @inline(__always)
   internal var zeroTerminatedRawCodeUnits: RawBitPattern {
-    @inline(__always) get {
-      return (
-        self._storage.0,
-        self._storage.1 & _StringObject.Nibbles.largeAddressMask)
-    }
+    return (self._storage.0, self._storage.1 & 0x00FF_FFFF_FFFF_FFFF)
```

Comment:
Losing the nice explanatory name on the mask here makes me a bit sad

Reply:
Right, but that name was misleading and conflating. This is a mask only for the small-string contents, which no longer aligns with addresses.

I thought about adding a static property for this, which would be inlinable and in our ABI, and only used in this one place. I'm not sure how to weigh the "at least it has an name" with "one level of cognitive indirection" if there's only one use site.

Reply:
A local name could help readability here. `let smallStringCodeUnitsMask = 0x00FF_FFFF_FFFF_FFFF` or whatever. An actual static property is probably overkill.